cmd/arc_summary/arc_summary.py

```diff
@@ -874,13 +874,21 @@ def _tunable_summary(Kstat):
     descriptions = {}
 
     if show_tunable_descriptions:
+
+        command = ["/sbin/modinfo", "zfs", "-0"]
+
         try:
-            command = ["/sbin/modinfo", "zfs", "-0"]
             p = Popen(command, stdin=PIPE, stdout=PIPE,
-                      stderr=PIPE, shell=False, close_fds=True)
+                    stderr=PIPE, shell=False, close_fds=True)
```

Comment:
@scotws Looks like you need a couple more spaces.

Reply:
@dinatale2 Ye gods ... next attempt ...